### ISSUE TITLE
May we propose usd/usdz exporting feature?

### ISSUE THREAD

**Issue by iqch** [microsoft/TRELLIS#91]
Since Kaoling distribution brings openusd anyway - it is just minimal efforts to implement usd/usdz export feature. We spent about 2 hours doing this, which allows to get results anywhere in modern DCC tools. With mesh export with texture generation and raw/processed pointcloud.
Thank you.

**Comment by TheSeanLavery** [comment]
Are you working on that?

**Comment by colortelevision** [comment]
usdz would be tight

**Comment by iqch** [comment]
Yes. We implemented it easily with python, since USD API is very simple. One trick was to figure that usdz is a just uncompressed zip archive file. When we will get permission to publish our solution example - we will do it.
